handsontable/src/core/focusCatcher/index.js

```diff
@@ -3,8 +3,8 @@ import { installFocusDetector } from './focusDetector';
 import {
   normalizeCoordsIfNeeded,
   getMostTopStartPosition,
-  getMostBottomEndPosition,
 } from './utils';
+import { getMostBottomEndPosition } from '../../helpers/mixed';
```

Comment:
For consistency's sake, may I suggest also moving the `getMostTopStartPosition` function to that helper?